gesturelibrary/include/single/singleFingerTap.h

```diff
@@ -11,6 +11,10 @@
 #define TAP_DISTANCE 100
 #endif
 
+#ifndef TAP_TIME
+#define TAP_TIME 7
```

Comment:
Please use params existing or define new params in gestureparams.h

Reply:
Fixed